Microsoft.Toolkit.Uwp.UI.Animations/Extensions/AnimationExtensions.Light.cs

```diff
@@ -67,20 +68,22 @@ public static bool IsLightingSupported
         /// <param name="distance">The value.</param>
         /// <param name="duration">The duration.</param>
         /// <param name="delay">The delay.</param>
+        /// <param name="color">The color of the spotlight.</param>
         /// <returns>An animation set.</returns>
         public static AnimationSet Light(
             this FrameworkElement associatedObject,
             double distance = 0d,
             double duration = 500d,
-            double delay = 0d)
+            double delay = 0d,
+            Color color = default(Color))
```

Comment:
By default, shouldn't it be Colors.White instead of Black?

Reply:
good spot, I changed the type to be nullable, so if it detects null it will change it to White. You can't set the Color object as a default parameter as it needs to be a compile time constant.